app.rb

```diff
@@ -0,0 +1,154 @@
+  include LoadData
+  def initialize
+    @games = []
+    @authors = []
+    @books = []
+    @labels = []
+    @genres = []
+    @albums = []
+  end
+
+  def list_all_games
+    if @games.empty?
+      puts 'No games available'
+    else
+      @games.each do |game|
+        puts "ID:#{game.id}
+              Published_date:#{game.published_date}
+              Multiplayer:#{game.multiplayer}
+              Last_played:#{game.last_played_at}"
+      end
+    end
+  end
+
+  def list_all_authors
+    if @authors.empty?
+      puts 'There are no authors'
+    else
+      @authors.each do |author|
+        puts "Author: #{author.first_name} #{author.last_name}"
+      end
+    end
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each do |book|
+        puts "ID:#{book.id}
+              Published_date:#{book.published_date}
+              Publisher:#{book.publisher}
+              Cover_state:#{book.cover_state}"
+      end
+    end
+  end
+
+  def list_all_labels
+    if @labels.empty?
+      puts 'No labels available'
+    else
+      @labels.each do |label|
+        puts "ID:#{label.id}
+              Title:#{label.title}
+              Color:#{label.color}"
+      end
+    end
+  end
+
+  def list_all_music_albums
+    if @music_album.nil? || @music_album.empty?
+      puts 'No music albums available'
+    else
+      @music_album.each do |music_album|
+        puts "ID: #{music_album.id}"
+        puts "Release Date: #{music_album.published_date}"
+        puts "Album Name: #{music_album.name}"
+        puts '-----------------------'
+      end
+    end
+  end
+
+  def list_all_genres
+    if @genres.empty?
+      puts 'No genres available'
+    else
+      @genres.each do |genre|
+        puts "ID: #{genre.id}"
+        puts "Name: #{genre.name}"
+        puts '-----------------------'
+      end
+    end
+  end
+
+  def add_game
+    published_date, archived, multiplayer, last_played_at = user_input(['Enter published date eg "2020-02-05"',
+                                                                        'Enter archirved [True/False]',
+                                                                        'Enter multiplayer [True/False]',
+                                                                        'Enter last played date eg "2020-02-05"'])
+
+    game = Game.new(published_date, archived, multiplayer, last_played_at)
+    @games << game
+    puts 'Game added successfully'
+  end
+
+  def add_book
+    publisher, cover_state, published_date, archived = user_input(['Enter publisher',
+                                                                   'Enter cover state eg [good/bad]',
+                                                                   'Enter published date eg "2020-02-05"',
+                                                                   'Enter archirved [True/False]'])
+
+    book = Book.new(publisher, cover_state, published_date, archived)
```

Comment:
- [ ] Let's make our app work even better here, what should make an item being archived should not be based on the user option, but the information entered by that user. For example, if the cover state is bad or the published date is older than 10 years then the can_be_archived function should return true.